Sources/CRoaring/roaring.c

```diff
@@ -16603,6 +16603,12 @@ roaring_uint32_iterator_t *roaring_create_iterator(const roaring_bitmap_t *r) {
     return newit;
 }
 
+roaring_uint32_iterator_t roaring_create_iterator_value(const roaring_bitmap_t *r) {
```

Comment:
We don't want to modify roaring.c and roaring.h, as it is an upstream dependency.